tests/sample/test_sample_models.py

```diff
@@ -0,0 +1,205 @@
+from datetime import datetime
+import uuid
+
+from dataservice.extensions import db
+from dataservice.api.participant.models import Participant
+from dataservice.api.sample.models import Sample
+from tests.utils import FlaskTestCase
+
+from sqlalchemy.exc import IntegrityError
+
+class ModelTest(FlaskTestCase):
+    """
```

Comment:
I know this is nitpicking, but could you add docstrings to all of the methods 😬

Reply:
👍